src/Math-Matrix/PMMatrix.class.st

```diff
@@ -311,6 +311,12 @@ PMMatrix >> atColumn: aColumnNumber put: aCollection startingAt: rowNumber [
 
 ]
 
+{ #category : #'cell accessing' }
```

Comment:
Any programmer tests to write here, @rakki-18 ? I seems simple enough

Reply:
@hemalvarambhia Yeah sure, I will write some tests.

Reply:
> @hemalvarambhia Yeah sure, I will write some tests.

TYSM